sdk/spring/azure-spring-cloud-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/servicebus/AzureServiceBusQueueAutoConfigurationTest.java

```diff
@@ -94,6 +97,24 @@ public void testResourceManagerProvided() {
                           });
     }
 
+    @Test
+    public void testMessageConverterProvided() {
+        this.contextRunner.withUserConfiguration(
+            TestConfigWithMessageConverter.class,
```

Comment:
Do we need to test this case: 
```
        this.contextRunner.withUserConfiguration(
            AzureServiceBusAutoConfiguration.class)
                          .withPropertyValues(
                              SERVICE_BUS_PROPERTY_PREFIX + "connection-string=str1"
                          )
        ...
```

Reply:
Hi, do you mean add a test specifically for connection-string? But it seems not a prerequisite of the autoconfiguration. So do we need to test it?